openmdao/visualization/n2_viewer/n2_viewer.py

```diff
@@ -378,7 +381,26 @@ def _get_viewer_data(data_source):
             return {}
 
     elif isinstance(data_source, str):
-        data_dict = CaseReader(data_source, pre_load=False).problem_metadata
+        cr = CaseReader(data_source, pre_load=True)
```

Comment:
Do we need to load *all* of the cases here using per_load=True?  Seems like we're wasting a lot of memory.